src/fides/api/models/privacy_preference.py

```diff
@@ -153,6 +166,12 @@ def __tablename__(self) -> str:
 
     fides_string = Column(String)
 
+    property_id = Column(
```

Comment:
Was this just added to get the privacy center version of the Get preferences endpoint working? (Do you think this change makes sense, outside of my specific request for just that one endpoint? Just wanted to understand your POV here, not necessarily disagreeing).